Makefile

```diff
@@ -125,11 +125,11 @@ flake8: ## Lints all Python files with flake8
 		| perl -F':\s+' -nE '$$F[1] =~ m/text\/x-python/ and say $$F[0]' \
 		| xargs flake8
 
-update-fedora-templates: assert-dom0 ## Upgrade Fedora 25 to Fedora 26 templates
-	sudo qubes-dom0-update qubes-template-fedora-26
-	sudo qubes-prefs default_template fedora-26
+update-fedora-templates: assert-dom0 ## Upgrade to Fedora 28 templates
+	sudo qubes-dom0-update qubes-template-fedora-28
+	sudo qubes-prefs default_template fedora-28
 	sudo qubesctl state.sls qvm.default-dispvm
-	qubes-prefs default_dispvm fedora-26-dvm
+	qubes-prefs default_dispvm fedora-28-dvm
```

Comment:
I get a qubes-prefs error on this line: `No such domain \'fedora-28-dvm\'`
It seems like we need to create a fedora-28 dvm manually from the template (as I believe fedora-26-dvm was included by default in the Qubes R4 install). The following two commands should do the trick [0] :
```
qvm-create --template fedora-28 --label red fedora-28-dvm
qvm-features fedora-28-dvm appmenus-dispvm 1
```

[0] : https://www.qubes-os.org/doc/dispvm-customization/